input/fsh/PatientIdentifierPatternConformanceSE.fsh

```diff
@@ -0,0 +1,29 @@
+Profile: PatientIdentifierPatternConformanceSE
+Parent: PatientSEVendorLite
+Id: PatientIdentifierPatternConformanceSE
+Description: "This profile enables format limitations to be put on the identifier in order to comply with Swedish legislation."
+* ^url = "http://care.commonprofiles.fhir/StructureDefinition/PatientIdentifierPatternConformanceSE"
+* ^version = "1.0.0"

```

Comment:
We should have a discussion if we want to version manage on profile level as well as IG level.

Reply:
Without having given it much thought, versioning all artefacts _seems_ like a reasonable thing to do. 

What are the cons besides more things to manage?